### PR TITLE
User can now view and scroll through the carousel

### PR DESCRIPTION
**Pull request by aaronkeith**
screen shot 2017-06-13 at 12 43 36 pm
screen shot 2017-06-13 at 12 43 52 pm
screen shot 2017-06-13 at 12 43 58 pm

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  color: #83fe00;
+  text-decoration: none;
+}
+/* ---Media Queries------ */
+
+@media (max-width: 549px) {
+  #myCarousel .item { height: 310px; }
+
+  #slide1 {
+    background: url('images/carousel_xsmall_01_run_opt.png') top center no-repeat;
```
Comment (thebearingedge) [inline styles.css]: See if you can extract the background positioning into its own class.
https://developer.mozilla.org/en-US/docs/Web/CSS/background?v=control

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Double-check your indentation. I'm seeing this:
https://raw.githubusercontent.com/aaronkeith/swoosh-shop/79cfe213a3ba95719bfbff025243f1e39ab2d02f/index.html

Your carousel html is wildly different. We want to stick with 2-space (soft tab) indentation.

**Comment by aaronkeith** [comment]
Whoa, that is wild.  This is what it looks like in my editor.
screen shot 2017-06-13 at 8 48 55 pm

**Comment by thebearingedge** [comment]
Yeah, your editor is probably not configured to re-write the tabs to spaces or something.

**Pull request merged**